src/codec/Utils.ts

```diff
@@ -23,7 +23,7 @@ export class Utils {
             size += 1;
         }
 
-        size += value == null ? 0 : value.length;
+        size += value == null ? 0 : Buffer.byteLength(value, 'utf8');
```

Comment:
Is this `Utils` class used anywhere? It duplicates `BitsUtil`'s functionality. Maybe we should remove it?

Reply:
Thanks for the remark, this file is duplicate and unused as you said. I removed it.